pkg/ebpf/tracee.go

```diff
@@ -862,6 +890,104 @@ func (t *Tracee) newConfig(cfg *policy.PoliciesConfig, version uint16) *Config {
 	}
 }
 
+func (t *Tracee) initKsymTableRequiredSyms() error {
```

Comment:
We need to make sure we always initialize the text segment in relevant events. Perhaps we may want it as a symbol always present. WDYT?

Reply:
Shouldn't this be handled by event dependencies?

Reply:
It should, that's why the first part was about double-checking that.